packages/tasks/src/model-libraries.ts

```diff
@@ -263,6 +263,22 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: { term: { path: "model.safetensors" } },
 	},
+	multi_token_prediction: {
+		prettyLabel: "Multi-Token Prediction",
+		repoName: "multi-token-prediction",
+		repoUrl: "https://huggingface.co/facebook/multi-token-prediction",
+		filter: false,
+		countDownloads: {
+			terms: {
+				path: [
+					"7B_1T_1/consolidated.pth",
+					"7B_1T_4/consolidated.pth",
+					"7B_200B_1/consolidated.pth",
+					"7B_200B_4/consolidated.pth",
+				],
+			},
```

Comment:
The format to define download count rules has changed. You should update your fork with the latest changes and modify this part. See for example [grok implementation](https://github.com/huggingface/huggingface.js/blob/22718122394af0165c55dd04cc4d3dc6ce547c17/packages/tasks/src/model-libraries.ts#L230)